src/controllers/highlights/HighlightPhrase.hpp

```diff
@@ -132,6 +132,15 @@ struct Deserialize<chatterino::HighlightPhrase> {
         chatterino::rj::getSafe(value, "color", encodedColor);
 
         auto _color = QColor(encodedColor);
+        if (!_color.isValid())
+        {
+            /*
+             * As a fallback, use the default self-highlight color. We can't use
+             * getApp()->themes here because the Application instance is not
+             * initialized when this function is called.
+             */
+            _color = QColor(127, 63, 73, 127);
```

Comment:
Why not either
- remove the color from Theme.cpp
- or use the value from Theme.cpp here

Reply:
> * or use the value from Theme.cpp here

I'm not sure how I would go about that - `getApp()->themes.messages.backgrounds.highlighted` wouldn't work here because the `Theme` singleton has not been initialized yet when this function is called.

Please let me know if I'm being dank and missing something obvious here.



> * remove the color from Theme.cpp

Yeah, I guess that would work. Since the highlight color is theme-independent now, this may be the way to go?

Reply:
> remove the color from Theme.cpp

I went ahead and did just that in 92e211a9. Let me know if you want that changed, I'm resolving this convo for now.